app/models/null_profile.rb

```diff
@@ -1,14 +1,14 @@
 class NullProfile
   def address
-    ""
+    ''
```

Comment:
I think the unofficial rule is double quotes everywhere but I leave that up to @ansonK to decide